controller.py

```diff
@@ -173,7 +174,6 @@ def __init__(self, app, emailer_class=Emailer):
         super(LibraryRegistryController, self).__init__(app)
         self.annotator = LibraryRegistryAnnotator(app)
         self.log = self.app.log
-        self.library_list_cache = dict()
```

Comment:
This was the existing cache that was being used for the admin interface and not the mobile clients?

Reply:
Yeah. I meant it to be used by the mobile clients but I put it in the wrong method, and now it won't work anyway.